README.md

```diff
@@ -1,36 +1,47 @@
+For voracious readers, the internet can be a blessing and a curse. There are so many sites out there, with so much great content, so many links to click, so many recommendations to follow. It's easy to end up with a seemingly endless collection of open browser tabs, and no time to sit at your computer to read them all. 
 
-Enter ReadMe: an offline-first moble app that—together with its [companion bowser extension](https://github.com/Project-Readme/ReadMe-browser-extension "extension repository")—allows readers to save articles, blog posts, and other text from their desktops and retrive that content later for reading on their mobile device. 
+Enter ReadMe: an offline-first mobile app that—together with its [companion browser extension](https://github.com/Project-Readme/ReadMe-browser-extension "extension repository")—allows a reader to save articles, blog posts, and other text from their desktops and retrieve that content later for reading on their mobile device. If a user does run out of things to read our app has recommendations for them on the home screen. 
 
-Articles are downloaded from the remote database and saved locally on users' devices for offline reading until they decide to delete them. 
-
-And if a user does run out of things to read, our app can help with that as well: the homepage of the app is pre-populated with most popular and most recent articles saved by everyone in ReadMe's community of users.
-
-The ReadMe App was built using React Native, Redux, and Cloud Firestore.  
+The ReadMe App was built using React Native, Redux, and Cloud Firestore.    
 
 ## Getting Started
 
-This app is available on the Expo(https://expo.io/ "Expo homepage") platform. After you have installed Expo onto your device you can run the app via this link (the easier option,) or by building your own copy. 
+This app is available on the [Expo](https://expo.io/ "Expo homepage") platform. After you have installed Expo onto your device you can run the app via this link (the easier option,) or by building your own copy.  
 
-To build you own copy, clone this repository to your local machine and install it with the [npm cli](https://docs.npmjs.com/cli/npm "npm cli documentation") by running `npm install`.
+To build you own copy, clone this repository to your local machine and install it with the [npm cli](https://docs.npmjs.com/cli/npm "npm cli documentation") by running `npm install`. 
 
-With Expo install, you can then run `npm start` to launch the app.
+With Expo installed, you can then run `npm start` to launch the app. 
 
+To log in to our app you can create an account or use the following credentials username: guest and password: password. 
```

Comment:
To log in to our app you can create an account, or use the following credentials:
> username: guest  
> password: password